contracts/interfaces/IERC1271Wallet.sol

```diff
@@ -0,0 +1,11 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import "../CaveatEnforcer.sol";
+
+interface IERC1271Wallet {
```

Comment:
I actually see Delegatable as a coherent alternative to 1271, but wouldn’t block inclusion, I just wouldn’t recommend someone use 1271 in a delegatable contract account. https://roamresearch.com/#/app/capabul/page/8kGfzKQEC

Reply:
@danfinlay ahhh that should not actually be in this PR.

Started working on the EIP1271 enforcer and was using the interface for testing. However, I decided against trying to to rush and include the code in this PR, so removed the files, but apparently left the interface by mistake.

Will remove during the next PR, unless it's to include the new enforcer ;)